src/argilla/client/feedback/schemas/metadata.py

```diff
@@ -98,6 +103,19 @@ def check_values(cls, terms_values: List[str], values: Dict[str, Any]) -> List[s
     def server_settings(self) -> Dict[str, Any]:
         return {"type": self.type, "values": self.values}
 
+    def _all_values_exist(self, introduced_value: str) -> None:
+        if introduced_value not in self.values:
+            raise ValueError(
+                f"Provided '{self.name}={introduced_value}' is not valid, only values in {self.values} are allowed."
+            )
```

Comment:
Is this a pydantic validator?

Reply:
Well it's just a class-method in the `pydantic.BaseModel` which is later used as a validator in the property `_pydantic_field_with_validator`